src/main/java/pages/ContactUsPage.java

```diff
@@ -12,6 +12,14 @@
     private int timeOut = 5;
 
     private By contactUsPageTitle = By.xpath("//h1[contains(text(),'Contact')]");
+    private By firstName = By.id("form-input-firstName");
+    private By lastName = By.id("form-input-lastName");
+    private By companyName = By.id ("form-input-companyName");
+    private By phone = By.id("form-input-homePhone");
+    private By email = By.id("form-input-email");
+    private By message = By.id("form-input-message");
+    private By sendButton = By.xpath("/html/body/div[2]/main/div/section/article/div[2]/form/fieldset/div[5]/button");
```

Comment:
This xpath is as fragile as they come. It starts from the bottom of the page and directs all the way to the button.

This definitely needs to change to a more robust locator.

Reply:
Because the website is bilingual I would refrain from using the text to build your locators.

Reply:
Shall we talk about your comments and resolve the conversation ?  In the first week of April?